scripts/stac/imagery/tests/conftest.py

```diff
@@ -21,3 +21,8 @@ def fake_collection_metadata() -> Iterator[CollectionMetadata]:
         "geographic_description": None,
     }
     yield collection_metadata
+
+
+@pytest.fixture
+def fake_linz_slug() -> str:
```

Comment:
Can this be declared only once? 
It is duplicated in the code base.

Reply:
You could also convert it to not be a fixture, then it can be imported from anywhere.

Reply:
moved `conftest.py` to parent folder